frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behaviors/unload-arriving-vehicles/simulated-region-overview-behavior-unload-arriving-vehicles.component.html

```diff
@@ -1 +1,51 @@
-<p>simulated-region-overview-behavior-unload-arriving-vehicles works!</p>
+<div class="h-100 mh-100 overflow-auto">
+    <h6>Aus eintreffenden Fahrzeugen aussteigen</h6>
+    <div
+        title="Zeit bis zur Ankunft"
+        class="input-group input-group-sm mb-3 ms-3"
+        style="width: 100px"
+    >
+        <label class="form-label">Aussteigedauer</label>
+        <input
+            #timeInput="ngModel"
+            [ngModel]="((unloadDuration$ | async) ?? 0) / 60 / 1000"
+            (appSaveOnTyping)="updateUnloadTime($event * 1000 * 60)"
+            min="0"
+            required
+            step="0.1"
+            type="number"
+            class="form-control form-control-sm d-inline-block no-validation"
+            placeholder="Dauer"
+            data-cy="unloadVehicleBehaviorPopupDurationInput"
+        />
+        <span class="input-group-text">Min</span>
+    </div>
+
+    <h6>Gerade aussteigend</h6>
+    <table
+        class="table table-striped"
+        *ngIf="(vehiclesStatus$ | async)?.length ?? 0; else noUnloadVehicles"
```

Comment:
```suggestion
        *ngIf="(vehiclesStatus$ | async)?.length; else noUnloadVehicles"
```

I think you don't need the ` ?? 0`, it works without it, too.

Or is there something I did not think about?